[2024-06-21T03:11:20.903+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-21T03:11:20.925+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: gic_main_dag.load manual__2024-06-21T03:10:34+00:00 [queued]>
[2024-06-21T03:11:20.934+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: gic_main_dag.load manual__2024-06-21T03:10:34+00:00 [queued]>
[2024-06-21T03:11:20.935+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 2
[2024-06-21T03:11:20.949+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): load> on 2024-06-21 03:10:34+00:00
[2024-06-21T03:11:20.961+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1479) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-21T03:11:20.963+0000] {standard_task_runner.py:63} INFO - Started process 1481 to run task
[2024-06-21T03:11:20.963+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'gic_main_dag', 'load', 'manual__2024-06-21T03:10:34+00:00', '--job-id', '210', '--raw', '--subdir', 'DAGS_FOLDER/main_dag.py', '--cfg-path', '/tmp/tmpv0xs98mw']
[2024-06-21T03:11:20.964+0000] {standard_task_runner.py:91} INFO - Job 210: Subtask load
[2024-06-21T03:11:21.036+0000] {task_command.py:426} INFO - Running <TaskInstance: gic_main_dag.load manual__2024-06-21T03:10:34+00:00 [running]> on host 39e4faefdcb7
[2024-06-21T03:11:21.195+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='***@example.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='gic_main_dag' AIRFLOW_CTX_TASK_ID='load' AIRFLOW_CTX_EXECUTION_DATE='2024-06-21T03:10:34+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-06-21T03:10:34+00:00'
[2024-06-21T03:11:21.199+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-21T03:11:21.277+0000] {logging_mixin.py:188} INFO - financial_type,symbol,security_name,sedol,price,quantity,realised_pl,market_value,fund_name,date,hash_uid,isin
[2024-06-21T03:11:21.335+0000] {logging_mixin.py:188} INFO - Connection successful.
[2024-06-21T03:11:21.387+0000] {logging_mixin.py:188} INFO - Error: invalid input syntax for type real: "price"
CONTEXT:  COPY raw_external_fund, line 1, column price: "price"
[2024-06-21T03:11:21.387+0000] {main_dag.py:140} ERROR - invalid input syntax for type real: "price"
CONTEXT:  COPY raw_external_fund, line 1, column price: "price"

[2024-06-21T03:11:21.389+0000] {python.py:237} INFO - Done. Returned value was: 1
[2024-06-21T03:11:21.389+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-21T03:11:21.413+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=gic_main_dag, task_id=load, run_id=manual__2024-06-21T03:10:34+00:00, execution_date=20240621T031034, start_date=20240621T031120, end_date=20240621T031121
[2024-06-21T03:11:21.458+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-21T03:11:21.482+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-21T03:11:21.486+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
